components/PageHeader.vue

```diff
@@ -34,62 +47,64 @@
           data-qa="logo"
         >
       </SmartLink>
-      <SearchForm
-        data-qa="search form"
-        role="search"
-        class="px-lg-3 mr-lg-auto mx-xl-auto"
-        aria-label="search form"
-        :enable-auto-suggest="enableAutoSuggest"
-      />
-    </header>
-    <b-navbar
-      class="p-lg-0 align-items-start justify-content-lg-end flex-column flex-lg-row d-none d-lg-block"
-      role="navigation"
-      data-qa="desktop navigation"
-    >
-      <PageNavigation
-        v-if="mainNavigation"
-        :links="mainNavigation.links"
-      />
-    </b-navbar>
-    <transition name="slide">
       <b-navbar
-        v-if="showSidebar"
-        class="p-lg-0 align-items-start justify-content-lg-end flex-column flex-lg-row d-lg-none"
+        class="align-items-center flex-row d-flex p-0 pr-3"
```

Comment:
I spotted a minor regression caused by pr-3, this caused the hamburger icon to be half width on Safari/iOS devices. I've pushed a fix for this.
![Screenshot 2020-10-09 at 19 34 30](https://user-images.githubusercontent.com/4670058/95573892-1b3fb000-0a67-11eb-8058-4f4556aa6744.png)